src/wallet/wallettool.cpp

```diff
@@ -68,23 +68,26 @@ static std::shared_ptr<CWallet> LoadWallet(const std::string& name, const fs::pa
     }
```

Comment:
When changing this file: please consider changing to const reference in `catch` on L65.